compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -933,6 +933,14 @@ impl<'ll> CodegenCx<'ll, '_> {
         ifn!("llvm.va_end", fn(ptr) -> void);
         ifn!("llvm.va_copy", fn(ptr, ptr) -> void);
 
+        // pointer authentication intrinsics
+        ifn!("llvm.ptrauth.sign.i64", fn(t_i64, t_i32, t_i64) -> t_i64);
+        ifn!("llvm.ptrauth.auth.i64", fn(t_i64, t_i32, t_i64) -> t_i64);
+        ifn!("llvm.ptrauth.strip.i64", fn(t_i64, t_i32) -> t_i64);
+        ifn!("llvm.ptrauth.resign.i64", fn(t_i64, t_i32, t_i64, t_i32, t_i64) -> t_i64);
+        ifn!("llvm.ptrauth.sign_generic.i64", fn(t_i64, t_i64) -> t_i64);
+        ifn!("llvm.ptrauth.blend.i64", fn(t_i64, t_i64) -> t_i64);
```

Comment:
You only need to add LLVM intrinsics in this place that are actually called by rustc_codegen_llvm.

Reply:
@arttet is this comment resolved?

thanks

Reply:
My sincere apologies for the delay.
So, I believe I've fixed it, is that correct?